components/tags.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
In your description, are you talking about the platformcomponents/desktop/tags.json?
What is different between desktop and web for this component then?

Reply:
After a quick look, they seem identical.

Reply:
I think you confused `tab` with `tag`?